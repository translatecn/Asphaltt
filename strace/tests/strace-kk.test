#!/bin/sh
#
# Ensure that strace -kk works.
#
# Copyright (c) 2023-2024 The strace developers.
# All rights reserved.
#
# SPDX-License-Identifier: GPL-2.0-or-later

KOPT_SHORT=-kk
KOPT_LONG=--stack-trace=source

. "${srcdir=.}"/strace-k.test
