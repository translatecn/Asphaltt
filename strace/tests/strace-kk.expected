chdir f3<stack-fcall-3.c:18> f2<stack-fcall-2.c:15> f1<stack-fcall-1.c:15> f0<stack-fcall-0.c:15> main<stack-fcall.c:25>
SIGURG f3<stack-fcall-3.c:21> f2<stack-fcall-2.c:15> f1<stack-fcall-1.c:15> f0<stack-fcall-0.c:15> main<stack-fcall.c:25>
